hedgehog/src/Hedgehog/Internal/Property.hs

```diff
@@ -809,6 +828,12 @@ evalEither = \case
   Right x ->
     pure x
 
+-- | Fails the test if the action throws an exception, or if the
+--   'Either' is 'Left'.  Otherwise returns the value in the 'Right'.
```

Comment:
```suggestion
--   'Either' is 'Left'. Otherwise returns the value in the 'Right'.
```

Nowadays, people usually don’t put two spaces after punctuation. (And, most importantly, other comments in this repository have only one space, as far as I can tell.)